docker-compose.yml

```diff
@@ -126,8 +126,8 @@ services:
       context: ./forms-flow-web/
       dockerfile: Dockerfile
       args:
-      - FORMIO_SOURCE_REPO_BRANCH=${FORMIO_SOURCE_REPO_BRANCH}
-      - FORMIO_SOURCE_REPO_URL=${FORMIO_SOURCE_REPO_URL}
+        - FORMIO_SOURCE_REPO_BRANCH=${FORMIO_SOURCE_REPO_BRANCH}
+        - FORMIO_SOURCE_REPO_URL=${FORMIO_SOURCE_REPO_URL}
```

Comment:
I turned off my "format on save" so hopefully don't see these formatting overhead in the review anymore :)